--- a/tool/src/s.js
+++ b/tool/src/s.js
@@ -20,11 +20,7 @@ function preload_boundaries(id) {
 };
 
 async function geography(c) {
-	const coll = await API.get("geographies", {
-		"datasets_count": "gt.0",
-		"parent_id":      `eq.${c.id}`,
-		"deployment":     `ov.{${ENV}}`,
-	});
+	const coll = [c, c.subgeographies].flat();
 
 	if (c.datasets_count > 2) coll.unshift(c); // 2 datasets: outline and admin-tiers
 
--- a/tool/src/a.js
+++ b/tool/src/a.js
@@ -146,6 +146,8 @@ async function init_1() {
 	const url = new URL(location);
 	const id = url.searchParams.get('id');
 
+	qs('#snapshot-button').remove();
+
 	GEOGRAPHY = await API.get("geographies", {
 		"id":     `eq.${id}`,
 		"select": ['*', 'parent_sort_branches', 'parent_sort_subbranches', 'parent_sort_datasets'],
@@ -176,8 +178,6 @@ On your OS, you can do this by pressing (${mac ? "⌘" : "ctrl"} −) a couple t
 
 	GEOGRAPHY.timeline_dates = maybe(GEOGRAPHY, 'configuration', 'timeline_dates');
 
-	session_init();
-
 	PARAMS = {
 		"view":    ['inputs', 'filtered', 'outputs'],
 		"output":  ['eai', 'ani', 'demand', 'supply'],
